xrdp/xrdp.ini.in

```diff
@@ -27,6 +27,10 @@ port=3389
 ; prefer use vsock://<cid>:<port> above
 use_vsock=false
```

Comment:
Btw, if this is not recommended, it could be removed from the .ini's default, while still being supported for backwards compatibility. I find it slightly confusing.